scripts/globals/moghouse.lua

```diff
@@ -27,8 +29,9 @@ function moogleTrade(player,npc,trade)
             end
         end
 
-        GiveMoogleABreak = player:getQuestStatus(OTHER_AREAS_LOG,GIVE_A_MOOGLE_A_BREAK)
-        if GiveMoogleABreak == QUEST_ACCEPTED and trade:hasItemQty(17161, 1) and trade:hasItemQty(13457, 1) then
+        local GiveMoogleABreak = player:getQuestStatus(OTHER_AREAS_LOG, dsp.quest.id.otherAreas.GIVE_A_MOOGLE_A_BREAK)
+        if GiveMoogleABreak == QUEST_ACCEPTED and npcUtil.tradeHas(trade, {17161, 13457}) then
+            player:confirmTrade()
```

Comment:
It's usually better to have t`confirmTrade`s in the `eventFinish`, to avoid situations where the event gets canceled and we took the player's items without advancing their variables.